hackathon_site/dashboard/frontend/src/components/dashboard/DashCard/DashCard.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+@import "./../../../assets/abstracts/variables";
+
+.DashCard {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    h2 {
+        margin-bottom: 10px;
+        display: flex;
+        justify-content: left;
+    }
+    .paper {
+        flex-grow: 1;
+        padding: 8px 0px;
+        overflow-y: scroll;
+        overflow-x: hidden;
+        .items {
+            display: flex;
+            flex-flow: row nowrap;
```

Comment:
can remove this since both `row` and `nowrap` are default

Reply:
Done!